packages/oui-clipboard/src/clipboard.html

```diff
@@ -0,0 +1,12 @@
+<div class="oui-clipboard">

```

Comment:
This class could be added on the root component `<oui-clipboard>`, to avoid this DIV.
See our use of `$postLink` for that :)